scaffolds/csharp/README.md

```diff
@@ -1,33 +1,36 @@
 # C# Scaffold for Code Retreat
 
+Workflow when using this scaffold:
+1. Start the Docker container with the tools needed to build, run and test your solution (see below).
+2. Edit your code using you favorite editor.
+3. Run the unit tests inside the container using ``dotnet test``.
+
 ## Start the .NET CLI and .NET Core
 
-Run a bash shell inside the docker container :
+Under Linux, run a bash shell inside the docker container from the terminal:
 
 ```Bash
 ./docker.sh
 ```
-## Run the code
 
-Once inside the docker container, you can do :
+Or, under Windows, run a bash shell inside the docker container from PowerShell:
 
-```Bash
-cd src/CodeRetreat
-dotnet restore
-dotnet run
```

Comment:
I haven't found any explanation on how to run the app, only how to test it. Could you please include some ? Thanks !

Reply:
The "app" as it stands is not supposed to be run, it's really just a library. Do you want me to add a console project to the solution so that it can be run?

Reply:
I think it would be nice, so that it can be useful in more situations !

Reply:
Done.